modules/about-cli-profiles-switch.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/managing-cli-profiles.adoc
+
+[id="about-switches-between-cli-profiles_{context}"]
+= About switches between CLI profiles
+
+Contexts allow you to easily switch between multiple users across multiple
```

Comment:
Looks like this section is hard wrapped at 80 characters. In 4.x, we updated our guidelines to no longer hard wrap lines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

Please check for hard wrapping across all files.

Reply:
Ok I am figuring out hard wrap, do i select a text and do a ctrl alt and y

Reply:
And this is only applicable for paragraphs?

Reply:
Is there a reason why we initially hard wrapped at 80 and no longer do that?

Reply:
Ok I applied ctrl+j to paras and lenthy bullet points as well